message_sender/serializers.py

```diff
@@ -48,20 +73,9 @@ class Meta:
         fields = (
             'url', 'id', 'message_id', 'reply_to', 'to', 'from_addr',
             'content', 'channel_id', 'channel_data', 'created_at',
-            'updated_at')
-
-    def to_internal_value(self, data):
-        """
-        Maps Junebug 'from' field to 'from_addr' expected by serializer since
-        'from' is a python keyword.
-        """
-        if "from" not in data:
-            raise serializers.ValidationError({
-                'from': 'This field is required.'
-            })
-        else:
-            data['from_addr'] = data['from']
```

Comment:
The create function (https://github.com/praekelt/seed-message-sender/blob/feature/MOMZA-610-identity-field-on-message-sender/message_sender/views.py#L143) is called before the serializer, so we have already removed `from`/`from_addr` and added `from_identity. Do we need it for anything else?